Fixed missing debugs char -> wchar_t

ross-weir opened the pull request:
Fixed some string instances that were missed when converting to wide chars.

Strings were only accessible in `Debug` builds

noah- commented:
the change looks okay to me
have you tested the debug build to make sure it works?

noah- commented on Script.cpp:
```diff
@@ -432,12 +432,12 @@ void Script::FireEvent(Event* evt) {
 typedef struct tagTHREADNAME_INFO {
     DWORD dwType;     // must be 0x1000
-    LPCSTR szName;    // pointer to name (in user addr space)
+    const wchar_t* szName;    // pointer to name (in user addr space)
```
you can use LPCWSTR for consistency

noah- commented on Script.cpp:
```diff
@@ -432,12 +432,12 @@ void Script::FireEvent(Event* evt) {
 } THREADNAME_INFO;
 
-void SetThreadName(DWORD dwThreadID, LPCSTR szThreadName) {
+void SetThreadName(DWORD dwThreadID, const wchar_t* szThreadName) {
```
you can use LPCWSTR for consistency

ross-weir commented:
> the change looks okay to me
> have you tested the debug build to make sure it works?

Debug build now compiles at least. But it fails to inject with:

`LoadLibraryA on remote process failed with error: 87`

I tried to debug it for a bit but haven't been able to figure out why. I know the debug build used to work fine.

Addressed feedback

noah- commented:
did you copy the js lib files in the debug folder resources?

noah- commented on Script.cpp:
```diff
@@ -432,12 +432,12 @@ void Script::FireEvent(Event* evt) {
 typedef struct tagTHREADNAME_INFO {
     DWORD dwType;     // must be 0x1000
-    const wchar_t* szName;    // pointer to name (in user addr space)
+    LPCWSTR szName;    // pointer to name (in user addr space)
```
can you align the comment
